File: src/main/java/club/geek66/querydsl/PropertyFilter.java
package club.geek66.querydsl;


import lombok.Builder;
import lombok.Data;
import lombok.RequiredArgsConstructor;

/**
 * @author: orange
 * @date: 2020/12/3
 * @time: 上午9:52
 * @copyright: Copyright 2020 by orange
 */
@Data
@Builder
@RequiredArgsConstructor
public class PropertyFilter {

	private final String property;

	private final String operator;

	private final Object value;

}

File: src/main/kotlin/club/geek66/querydsl/PropertyFilter.kt
package club.geek66.querydsl

/**
 * @author: orange
 * @date: 2020/12/3
 * @time: 上午9:52
 * @copyright: Copyright 2020 by orange
 */
data class PropertyFilter(
	val property: String,
	val operator: String,
	val value: Any? = null
)